micrometer-core/build.gradle

```diff
@@ -10,6 +10,8 @@ apply plugin: 'nebula.optional-base'
 dependencies {
     compile 'org.hdrhistogram:HdrHistogram:2.1.10'
     compile 'org.latencyutils:LatencyUtils:2.0.3'
+    compile 'io.opentracing:opentracing-api:0.30.0'
```

Comment:
hmm adding a core dependency? probably there's another way. (ps 0.31 is api breaking)